$NetBSD: patch-Makefile,v 1.1 2021/10/08 11:55:42 abs Exp $

Ensure LDFLAGS are passed to allow RELRO builds to work

--- Makefile.orig	2001-05-22 23:51:48.000000000 +0000
+++ Makefile
@@ -1,3 +1,3 @@
 all:
-	${CC} ${CFLAGS} -o unix2dos unix2dos.c
+	${CC} ${CFLAGS} ${LDFLAGS} -o unix2dos unix2dos.c
 	ln -f unix2dos dos2unix
